%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% "ModernCV" CV and Cover Letter
% LaTeX Template
% Version 1.1 (9/12/12)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Xavier Danaux (xdanaux@gmail.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
% Important note:
% This template requires the moderncv.cls and .sty files to be in the same
% directory as this .tex file. These files provide the resume style and themes
% used for structuring the document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%   PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[11pt,a4paper,roman]{moderncv} % Font sizes: 10, 11, or 12; paper sizes: a4paper, letterpaper, a5paper, legalpaper, executivepaper or landscape; font families: sans or roman


\moderncvstyle{classic} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{black} % CV color - options include: 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'

\usepackage{lipsum} % Used for inserting dummy 'Lorem ipsum' text into the template

\usepackage[scale=0.90]{geometry} % Control document margins
\setlength{\hintscolumnwidth}{3cm} % Change the width of the dates column
%\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name

%\usepackage[english]{babel}
%\usepackage{hyperref}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{color}
\usepackage[english]{babel}
\usepackage{etaremune}  % for reverse enumeration
\usepackage{enumitem} % for different style in itemize

% footer page count
\usepackage{lastpage}
\usepackage{fancyhdr}
\pagestyle{fancy}

\cfoot{\thepage\ of \pageref{LastPage}}

%----------------------------------------------------------------------------------------
%   NAME AND CONTACT INFORMATION SECTION
%----------------------------------------------------------------------------------------

\firstname{André} % Your first name
\familyname{F. Rendeiro} % Your last name

% All information in this block is optional, comment out any lines you don't need
\title{Computational biologist}
%\address{}
%\mobile{}
%\phone{}
%\fax{}
\email{andre.rendeiro@gmail.com}
\homepage{andre-rendeiro.com} % The first argument is the url for the clickable link, the second argument is the url displayed in the template - this allows special characters to be displayed such as the tilde in this example

\social[orcid][orcid.org/0000-0001-9362-5373]{ORCID: 0000-0001-9362-5373}
\social[github]{afrendeiro}
\social[twitter]{afrendeiro}


%\extrainfo{additional information}
%\photo[70pt][0.4pt]{/home/afr/Pictures/me.png} % The first bracket is the picture height, the second is the thickness of the frame around the picture (0pt for no frame)
%\quote{"A witty and playful quotation" - John Smith}

%----------------------------------------------------------------------------------------

\begin{document}

\makecvtitle % Print the CV title

%----------------------------------------------------------------------------------------
%   CURRENT POSITION
%----------------------------------------------------------------------------------------

\section{Current position}
    \cventry{03/2020 - present}
    {Postdoctoral Associate in Computational Biomedicine}{Institute of Computational Biomedicine, Englander Institute for Precision Medicine, Weill Cornell Medicine}{USA}
    {}{Supervisor: Olivier Elemento}

%----------------------------------------------------------------------------------------
%   EDUCATION SECTION
%----------------------------------------------------------------------------------------

\section{Education}
    \cventry{09/2014 - 01/2020}
        {PhD}{Medical University of Vienna}{Austria}{}{Supervisor: Christoph Bock, CeMM Research Centre for Molecular Medicine}
    \cventry{09/2012 - 06/2014}
        {'Mestrado' (MSc) in Molecular and Cell Biology}{University of Aveiro}{Portugal}{}{}
    \cventry{09/2008 - 07/2012}
        {'Licenciatura' (BSc) in Biology}{University of Aveiro}{Portugal}{}{}


%----------------------------------------------------------------------------------------
%   KEY PUBLICATIONS
%----------------------------------------------------------------------------------------


\section{Key publications}

    \cvitem{}
        {Kim \textit{et al.}, \textbf{Unsupervised discovery of tissue architecture in multiplexed imaging}. BioRxiv (2022) \href{https://dx.doi.org/10.1101/2022.03.15.484534}{doi:10.1101/2022.03.15.484534}}

    \cvitem{}
        {\underline{Rendeiro}*, Ravichandran* \textit{et al.}, \textbf{The spatial landscape of lung pathology during COVID-19 progression}. Nature (2021) \href{https://dx.doi.org/10.1038/s41586-021-03475-6}{doi:10.1038/s41586-021-03475-6}}
    \cvitem{}
        {\underline{Rendeiro}*, Krausgruber* \textit{et al.}, \textbf{Chromatin mapping and single-cell immune profiling define the temporal dynamics of ibrutinib drug response in CLL}. Nature Communications (2020). \href{https://dx.doi.org/10.1038/s41467-019-14081-6}{doi:10.1038/s41467-019-14081-6}}
    \cvitem{}{
        Datlinger*, \underline{Rendeiro}*, \textit{et al}. \textbf{Ultra-high throughput single-cell RNA sequencing by combinatorial fluidic indexing}. Nature Methods (2021). \href{https://dx.doi.org/10.1038/s41592-021-01153-z}{doi:10.1038/s41592-021-01153-z}
    }
    \cvitem{}{
        Datlinger, \underline{Rendeiro}*, Schmidl* \textit{et al.}, \textbf{Pooled CRISPR screening with single-cell transcriptome readout}. Nature Methods (2017) \href{https://dx.doi.org/10.1038/nmeth.4177}{doi:10.1038/nmeth.4177}
    }
    \cvitem{}{
        \underline{Rendeiro}*, Schmidl*, Strefford* \textit{et al.}, \textbf{Chromatin accessibility maps of chronic lymphocytic leukaemia identify subtype-specific epigenome signatures and transcription regulatory networks}. Nature Communications (2016). \href{https://dx.doi.org/10.1038/ncomms11938}{doi:10.1038/ncomms11938}
    }
    \cvitem{}{
        * \textit{equal first-author contributions}; $^\Omega$ \textit{joint corresponding authors}
    }

%----------------------------------------------------------------------------------------
%   AWARDS SECTION
%----------------------------------------------------------------------------------------

\section{Scholarships/Grants}
    \cventry{10/2020 - present}
        {Molecular and Translational Oncology Research}{Training Grant}{T32CA203702}{}{National Cancer Institute, USA}
    \cventry{08/2013 - 06/2014}
        {Erasmus studies mobility program scholarship}{}{}{}{European Commission}
    \cventry{09/2011 - 06/2012}
        {Erasmus intership mobility program scholarship}{}{}{}{European Commission}
    \cventry{10/2009 - 06/2010}
        {"Integration into Research" Grant}{}{}{}{Science and Technology Foundation, Portugal}

%----------------------------------------------------------------------------------------
%   PUBLICATIONS
%----------------------------------------------------------------------------------------
\newpage

\section{All publications}
    \cvitem{}{
        * \textit{equal first-author contributions}
        $^\Omega$ \textit{joint corresponding authors}
    }

    \cvitem{\underline{Preprints}}{(does not include preprints later published in peer-reviewed journals)}
        \begin{etaremune}[leftmargin=1.0cm, itemindent=0pt, topsep=10pt, itemsep=2pt, partopsep=0pt, parsep=0pt]

        {{preprints_go_here}}

        \end{etaremune}

    \cvitem{\underline{Peer reviewed publications}}{}
        \begin{etaremune}[leftmargin=1.0cm, itemindent=0pt, topsep=10pt, itemsep=2pt, partopsep=0pt, parsep=0pt]

        {{publications_go_here}}

        \end{etaremune}

\section{Communications}
    \cvitem{\underline{Conference talks}}{}
        \begin{etaremune}[leftmargin=1.0cm,itemindent=0pt,topsep=10pt,itemsep=2pt,partopsep=0pt,parsep=0pt]
        \item
        \textbf{The spatial landscape of lung pathology during COVID-19 progression}. \textit{IMC Summit, organized by Fluidigm}, October 2021.
        \item
        \textbf{Chromatin mapping and single-cell immune profiling define the temporal dynamics of Ibrutinib response in CLL}. \textit{Young Scientist Association of the Medical University of Vienna PhD Symposia}, June 2019, Vienna, Austria.
        \item
        \textbf{Chromatin mapping and single-cell immune profiling define the temporal dynamics of Ibrutinib response in CLL}. \textit{Frontiers in Single Cell Genomics Meeting - Cold Spring Harbour Asia}, November 2018, Suzhou, China.
        \item
        \textbf{CROP-seq: updates on the single cell CRISPR screening method}. \textit{10X User Group Meeting 2018}, April 2018, EMBL, Heidelberg, Germany.
        \item
        \textbf{Pooled CRISPR screening with single-cell transcriptome readout}. \textit{SLAS 2018}, February 2018, San Diego, USA.
        %\item
        \textbf{Pooled CRISPR screening with single-cell transcriptome readout}. \textit{Illumina User Group Meeting 2017}, February 2018, Bern, Switzerland.
        \item
        \textbf{Large-scale ATAC-seq profiling to identify disease subtypes, regulatory networks and monitoring treatment in CLL}. \textit{Illumina User Group Meeting 2017}, February 2018, Cologne, Germany.
        \item
        %Paul Datlinger, \underline{André F. Rendeiro}*, Christian Schmidl*, Thomas Krausgruber, Peter Traxler, Johanna Klughammer, Linda C Schuster, Amelie Kuchler, Donat Alpar, Christoph Bock.
        \textbf{Pooled CRISPR screening with single-cell transcriptome readout}. \textit{Ascona Workshop 2017}, May 2017, Ascona, Switzerland.
        \item
        %Michaela Schwaiger, Anna Schönauer, \underline{André F. Rendeiro}, Carina Pribitzer, Alexandra Schauer, Anna Gilles, Johannes Schinko, David Fredman, and Ulrich Technau.
        \textbf{Evolutionary conservation of the eumetazoan gene regulatory landscape}. \textit{XVIII Portuguese Genetics Society Meeting}, June 2013. Porto, Portugal
        \end{etaremune}

    \cvitem{\underline{Conference posters}}{}
        \begin{etaremune}[leftmargin=1.0cm,itemindent=0pt,topsep=10pt,itemsep=2pt,partopsep=0pt,parsep=0pt]

        \item
        %\underline{André F. Rendeiro}*, Thomas Krausgruber*, Nikolaus Fortelny, Fangwen Zhao, Thomas Penz, Matthias Farlik, Linda C. Schuster, Amelie Nemc, Szabolcs Tasnády, Marienn Réti, Zoltán Mátrai, Donat Alpar, Csaba Bödör, Christian Schmidl, Christoph Bock.
        \textbf{Chromatin mapping and single-cell immune profiling define the temporal dynamics of ibrutinib drug response in chronic lymphocytic leukemia}. \textit{SCOG Workshop Computational Single Cell Genomics}, May 2019. Munich, Germany. \href{https://doi.org/10.6084/m9.figshare.7892663.v1}{doi:10.6084/m9.figshare.7892663.v1}
        \item
        %Christian Schmidl*, \underline{André F. Rendeiro}*, Gregory I Vladimer*, Thomas Krausgruber, Tea Pemovska, Nikolaus Krall, Berend Snijder, Oscar Lopez de la Fuente, Anna Ringler, Stefan Kubicek, Philipp B. Staber, Medhat Shehata, Giulio Superti-Furga, Ulrich Jäger, Christoph Bock.
        \textbf{Combined chromatin accessibility and chemosensitivity profiling identifies targetable pathways and rational drug combinations in Ibrutinib-treated chronic lymphocytic leukemia}. \textit{Young Scientist Association of the Medical University of Vienna PhD Symposia}, June 2017. Vienna, Austria.
        \item
        %\underline{André F. Rendeiro}*, Christian Schmidl*, Jonathan C. Strefford*, Renata Walewska, Zadie Davis, Matthias Farlik, David Oscier, Christoph Bock.
        \textbf{Large-scale chromatin profiling uncovers heterogeneity of molecular phenotypes and gene regulatory networks of chronic lymphocytic leukemia}. \textit{Young Scientist Association of the Medical University of Vienna PhD Symposia}, June 2016, Vienna, Austria. \href{https://doi.org/10.6084/m9.figshare.3479528.v1}{10.6084/m9.figshare.3479528.v1} \textbf{Best poster award in "Malignant Diseases" category}.
        \item
        %\underline{André F. Rendeiro}*, Christian Schmidl*, Jonathan C. Strefford*, Renata Walewska, Zadie Davis, Matthias Farlik, David Oscier, Christoph Bock.
        \textbf{Large-scale chromatin profiling uncovers heterogeneity of molecular phenotypes and gene regulatory networks of chronic lymphocytic leukemia}. \textit{Keystone Symposia on Chromatin and Epigenetics}, March 2016, Whistler, Vancouver, Canada. https://doi.org/10.6084/m9.figshare.3479528.v1
        \item
        %Anna Schönauer, \underline{André F. Rendeiro}, Michaela Schwaiger, Ulrich Technau.
        \textbf{Identification of cis-regulatory elements in the sea anemone \textit{Nematostella vectensis}}. \textit{Evonet Symposium}, September 2012, Vienna Austria. \href{https://doi.org/10.6084/m9.figshare.107026}{doi:10.6084/m9.figshare.107026}
        \end{etaremune}

%----------------------------------------------------------------------------------------
%   WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\section{Additional experience}

    \subsection{\underline{Research Activity}}

        \cventry{08/2013 - 06/2014}
            {The role of E2F regulation and H3K79 methylation in \textit{Oikopleura dioica}'s cell cycle modes}
            {Sars International Centre for Marine Molecular Biology, Bergen, Norway}{}
            {}
            {Supervisor: Eric Thompson}
            %\{I investigated the molecular mechanisms of alternative cell cycle modes (particularly endocycles) in the chordate \textit{Oikopleura dioica} by performing ChIP-seq on transcription factors involved in cell cycle regulation (E2F). I also studied the role of H3K79me on cell cycle regulation through functional studies on its methyltransferase, Dot1.}

        \cventry{09/2011 - 07/2012}
            {Identification of cis-regulatory elements in \textit{Nematostella vectensis} using ChIP-seq}
            {Dept. of Molecular Evolution and Development, University of Vienna, Austria}{}
            {}
            {Supervisor: Ulrich Technau}
            %\{I performed ChIP-seq of chromatin modifications and other regulatory proteins over several developmental stages of \textit{Nematostella vectensis}, constructed a map of chromatin states and predicted cis-regulatory elements genome-wide. I also tested the function of some of these regions \textit{in vivo} in a reporter assay.}

        \cventry{09/2010 - 06/2011}
            {Tol2-mediated zebrafish transgenesis for studies in protein mistranslation}
            {RNA Biology Laboratory, Biology Department, University of Aveiro, Portugal}{}
            {}
            {Supervisor: Manuel Santos}
            %\{I created transgenic zebrafish that were used as a model for studies in neurodegeneration through protein aggregation. This was caused by increasing the level of translational error (mistranslation) during endogenous protein synthesis. I learned to build plasmid constructs, microinject them in zebrafish and screen for phenotypes.}

        \cventry{09/2009 - 06/2010}
            {Transciptome studies with microarrays in heat-shocked yeast}
            {RNA Biology Laboratory, Biology Department, University of Aveiro, Portugal}{}
            {}
            {Supervisor: Manuel Santos}
            %\{I was involved in the analysis of microarray expression data from yeast under various treatments. I learned to pre-process, normalise and explore data programmatically to detect significant differential gene expression, clustering genes and exploring their ontology across treatments.}

    %----------------------------------------------------------------------------------------
    %   COURSES
    %----------------------------------------------------------------------------------------
    \subsection{\underline{Courses and other scientific activity}}
        \cvitem{01/2022}{PI Crash Course: Skills for Future or New Lab Leaders - Columbia University, NY, USA}
        \cvitem{04/2021}{Probabilistic Modeling in Genomics, Virtual meeting - Cold Spring Harbor, NY, USA}
        \cvitem{09/2015}{Summer School on Machine Learning for Personalised Medicine - Marie Curie Initial Training Network, Manchester, UK}
        \cvitem{09/2012}{Scientific writing course - University of Aveiro, Portugal}

    %----------------------------------------------------------------------------------------
    %   ADMIN
    %----------------------------------------------------------------------------------------
    \subsection{\underline{Associative/Administrative}}
        \cvitem{09/2010 - 06/2012}{Member of the Biology department counsel, University of Aveiro, Portugal}
        \cvitem{09/2009 - 06/2011}{Member of the undergraduate Biology committee, University of Aveiro, Portugal}

%----------------------------------------------------------------------------------------
%   LICENSES & CERTIFICATIONS
%----------------------------------------------------------------------------------------

\section{Licenses and certifications}

    \cventry{04/2021 - 04/2025}
        {Biomedical Research Investigators and Key Personel}{\href{https://www.citiprogram.org/verify/?w9102e99f-9a51-41fc-993b-e50ba7dafc36-41194853}{Credential ID: 41194853}}{}{}{CITI Program}

    \cventry{04/2021 - 04/2025}
        {Good Clinical Practice}{\href{https://www.citiprogram.org/verify/?w50fab502-7953-4e4c-ac09-87d4aa204f2f-41194854}{Credential ID: 41194854}}{}{}{CITI Program}

    \cventry{12/2020 - 12-2025}
        {Responsible Conduct of Research}{}{}{}{Tri-Institutional program: MSK Cancer Center, Weill Cornell Medical College, Rockefeller University}

    \cventry{06/2020 - 01/2026}
        {Responsible Conduct of Research for Faculty}{}{}{}{Weill Cornell Medical College}


%----------------------------------------------------------------------------------------
%   AWARDS SECTION
%----------------------------------------------------------------------------------------

\section{Awards}
    \cventry{06/2016}
        {Best poster award - "Malignant diseases" category}{YSA Symposium}{}{}{Young Scientist Association of the Medical University of Vienna}
    \cventry{06/2016}
        {Best artwork award - "Illustrations and digital simulations" category}{ScienceArt Competition of the YSA Symposium}{}{}{Young Scientist Association of the Medical University of Vienna}

%----------------------------------------------------------------------------------------
%   SKILLS SECTION
%----------------------------------------------------------------------------------------
\section{Skills}

\subsection{\underline{Computational Biology}}
    \cvitem{Data science}{
        Development of data processing pipelines;
        Data-driven unsupervised analysis and visualisation;
        Statistical analysis;
        Application of unsupervised and supervised machine learning models;
        Application of Bayesian methods and probabilistic programming}
    \cvitem{Applications}{
        Analysis of:
        image data (IMC, IHC, IF, brightfield);
        single cell *-seq;
        bulk ATAC-/ChIP-/RNA-seq;
        CyTOF and flow cytometry;
        CRISPR screens}
    \cvitem{Programming}{
        Experienced in \textit{Python} and \textit{R} programming;
        Knowledge of \textit{Rust} programming
        \newline
        Competence in software development: version control, testing, continuous integration.}

\subsection{\underline{Molecular Biology}}
    \cvitem{Techniques}{
        Chromatin IP, ChIP-seq, NGS libraries, Western blotting, PCR, Cloning
    \newline
        Additional experience in:
        Chemical screening,
        Zebrafish and \textit{Nematostella} handling and microinjection,
        basic experience in immunohistochemistry,
        fluorescence and confocal microscopy}


%----------------------------------------------------------------------------------------
%   SOFTWARE
%----------------------------------------------------------------------------------------
\section{Software}
% \subsection{\underline{Data processing pipelines}}
% \subsection{}
    \cvitem{IMC}{
        A package for the analysis of imaging mass cytometry data:
        \newline
        \href{https://github.com/ElementoLab/imc}{https://github.com/ElementoLab/imc}
    }
    \cvitem{imcpipeline}{
        A pipeline for the preprocessing of imaging mass cytometry data:
        \newline
        \href{https://github.com/ElementoLab/imcpipeline}{https://github.com/ElementoLab/imcpipeline}
    }
    \cvitem{imctransfer}{
        Program for the robust, parallel transfer of raw IMC data between machines:
        \newline
        \href{https://github.com/ElementoLab/imctransfer}{https://github.com/ElementoLab/imctransfer}
    }
    \cvitem{page-enrichment}{
        A Python implementation of the Parametric Analysis of Gene Set Enrichment (PAGE):
        \newline
        \href{https://github.com/afrendeiro/page-enrichment}{https://github.com/afrendeiro/page-enrichment}
    }
    \cvitem{ngs-toolkit}{
        A toolkit for the analysis of NGS data:
        \newline
        \href{https://github.com/afrendeiro/toolkit}{https://github.com/afrendeiro/toolkit}
    }
    \cvitem{peppy}{
        A package to work with Portable Encapsulated Projects (PEP) in Python:
        \newline
        \href{https://github.com/pepkit/peppy}{https://github.com/pepkit/peppy}
    }
    \cvitem{looper}{
        A job controller for Portable Encapsulated Projects (PEP):
        \newline
        \href{https://github.com/pepkit/looper}{https://github.com/pepkit/looper}
    }
    \cvitem{open\_pipelines}{
        Pipelines for a variety of NGS data:
        \newline
        \href{https://github.com/epigen/open\_pipelines}{https://github.com/epigen/open\_pipelines}
    }
    
    % \cvitem{ngstk}{
    %     A collection of CLI tools for common bioinformatics work callable from Python:
    %     \newline
    %     \href{https://github.com/pepkit/ngstk}{https://github.com/pepkit/ngstk}
    % }
    
% \subsection{\underline{Publications}}
%     \cvitem{doi:x,y,z}{
%         \href{https://github.com/ElementoLab/covid-imc}{https://github.com/ElementoLab/covid-imc}
%         \href{https://github.com/ElementoLab/covid-imc-viz}{https://github.com/ElementoLab/covid-imc-viz}
%     }
%     \cvitem{doi:x,y,z}{
%         \href{https://github.com/ElementoLab/covid-imc}{https://github.com/ElementoLab/covid-imc}
%         \href{https://github.com/ElementoLab/covid-imc-viz}{https://github.com/ElementoLab/covid-imc-viz}
%     }


%----------------------------------------------------------------------------------------
%   MEMBERSHIPS
%----------------------------------------------------------------------------------------

%\section{Associations}
%\cvitem{2020-}{New York Academy of Sciences}
%\cvitem{2015-2018}{European Hematology Association (EHA)}

%----------------------------------------------------------------------------------------
%   LANGUAGES SECTION
%----------------------------------------------------------------------------------------

\section{Languages}

\cvitemwithcomment{Portuguese}{\textnormal{Native speaker}}{}
\cvitemwithcomment{English}{\textnormal{Very good}}{}
\cvitemwithcomment{Spanish}{\textnormal{Conversational}}{}
\cvitemwithcomment{German}{\textnormal{Basic}}{Basic words and phrases only}
\cvitemwithcomment{French}{\textnormal{Basic}}{Basic words and phrases only}

%----------------------------------------------------------------------------------------
%   INTERESTS SECTION
%----------------------------------------------------------------------------------------

%\section{Other interests}

%\cvlistdoubleitem{Ballroom dancing}{Cinema}
%\cvlistdoubleitem{Singing}{Opera}
%\cvlistdoubleitem{Choral conducting}{Piano playing}


%----------------------------------------------------------------------------------------
%   TIMESTAMP
%----------------------------------------------------------------------------------------
\bigskip
\textit{Updated on {{current_date}}}



%----------------------------------------------------------------------------------------
%   COVER LETTER
%----------------------------------------------------------------------------------------

% To remove the cover letter, comment out this entire block

% \clearpage

% \recipient{HR Departmnet}{Corporation\\123 Pleasant Lane\\12345 City, State} % Letter recipient
% \date{\today} % Letter date
% \opening{Dear Sir or Madam,} % Opening greeting
% \closing{Sincerely yours,} % Closing phrase
% \enclosure[Attached]{curriculum vit\ae{}} % List of enclosed documents

% \makelettertitle % Print letter title

% \lipsum[1-3] % Dummy text

% \makeletterclosing % Print letter signature

%----------------------------------------------------------------------------------------

\end{document}
